src/cdi_ndi/CMakeLists.txt

```diff
@@ -21,6 +21,18 @@ if( NOT DEFINED NDI_ROOT_DIR )
   get_filename_component(NDI_ROOT_DIR "${NDI_LIBRARY}" DIRECTORY CACHE)
 endif()
 
+# Location of gendir and gendir.all files.
+if( NOT DEFINED NDI_DATA_DIR OR NOT EXISTS "${NDI_DATA_DIR}" )
+  if( DEFINED ENV{NDI_GENDIR_PATH} AND EXISTS "$ENV{NDI_GENDIR_PATH}" )
+    get_filename_component( NDI_DATA_DIR "$ENV{NDI_GENDIR_PATH}" DIRECTORY )
+  elseif( EXISTS "${NDI_ROOT_DIR}/share/gendir" )
```

Comment:
Are we expecting a gendir or gendir.all file?

Reply:
That is the question of the year!  For production, NDI expects you to use `gendir`.  But as I understand it, your project's unit tests need data from `gendir.all`  I think `gendir` is required for all NDI installations, but `gendir.all` isn't.